packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/make.mk

```diff
@@ -0,0 +1,10 @@
+###############################################################################
+#
+# 
+#
+###############################################################################
```

Comment:
Why the empty header?

Reply:
This was added when the first ONL platform was created.
So the newer platforms all follow this empty header.